utils/transaction-parser.ts

```diff
@@ -237,6 +237,28 @@ function parseZetaInstruction(ix: PartiallyDecodedInstruction): Instruction {
       };
       break;
 
+    case "placeOrderV3":
+      // price: number;
+      // size: number;
+      // side: Side;
+      // orderType: orderType
+      // clientOrderId: number | undefined;
+      // tag: String | undefined;
+      let placeOrderV3Data = decodedIx.data as zetaTypes.placeOrderV3;
+      decodedIx.data = {
+        price: utils.convertNativeBNToDecimal(placeOrderV3Data.price),
+        size: utils.convertNativeLotSizeToDecimal(
+          placeOrderV3Data.size.toNumber()
+        ),
+        side: Object.keys(placeOrderV3Data.side)[0],
+        orderType: Object.keys(placeOrderV3Data.orderType)[0],
+        clientOrderId: placeOrderV3Data.clientOrderId
```

Comment:
Is this just handling the case whereby `clientOrderId` is null so as not to fail when calling `.toString()`?

Think the idiomatic way to handle this is via option chaining (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

```suggestion
        clientOrderId: placeOrderV3Data?.clientOrderId
```

Reply:
That's way nicer actually. That clientOrderId code was just copied from the placeOrderV2 parser above, but I've changed it to use `?.` syntax for both. Testing it locally and it seems fine:

Leaving clientOrderId and tag empty: `{
  data: {
    price: 80,
    size: 20,
    side: 'ask',
    orderType: 'limit',
    clientOrderId: undefined,
    tag: 'SDK'
  },
  name: 'placeOrderV3'
}`

Passing in an empty string tag: `{
  data: {
    price: 80,
    size: 20,
    side: 'ask',
    orderType: 'limit',
    clientOrderId: undefined,
    tag: ''
  },
  name: 'placeOrderV3'
}`

Passing in sensible values for both: `{
  data: {
    price: 80,
    size: 20,
    side: 'ask',
    orderType: 'limit',
    clientOrderId: '1234',
    tag: 'A1B'
  },
  name: 'placeOrderV3'
}`

Reply:
Why does tag fallback to 'SDK' for empty tags?

Reply:
Because the SDK defaults to "SDK" if tag is undefined :) I assume that most people manually using the SDK won't bother to set the tag, so I thought it'd be nice to be able to differentiate between them and someone calling the instructions manually

Happy to change the behaviour if you've got other thoughts